python/count.py

```diff
@@ -0,0 +1,18 @@
+## Optimized file reader/counter
```

Comment:
oh yeah I also rewrote this to perform much faster on large files/not bog down the system